UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/RNWRepository.cs

```diff
@@ -54,5 +54,20 @@ public async Task<List<string>> GetYears()
                                 .Select(p => p.DateTimeGroup.Year.ToString())
                                 .Distinct().ToListAsync();
         }
+
+        public async Task<List<RadioNavigationalWarningsData>> GetRadioNavigationalWarningsDataList()
+        {
+            return await (from rnwWarnings in _context.RadioNavigationalWarnings
+                     join warning in _context.WarningType on rnwWarnings.WarningType equals warning.Id
```

Comment:
Maybe clearer as `warningType`
```suggestion
                     join warningType in _context.WarningType on rnwWarnings.WarningType equals warningType.Id
```

Reply:
done required changes